staff/templates/staff_base.html

```diff
@@ -1,82 +1,211 @@
-{% load static %}
 <!DOCTYPE html>
-<html>
+{% load static %}
+{% load settings_tags %}
+<html lang="en">
 <head>
-  <title>Mission Bit</title>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
-  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link href="//cdn.muicss.com/mui-0.9.43/css/mui.min.css" rel="stylesheet" type="text/css" />
+    <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'>
+    <link rel="stylesheet" href="{% static  'css/mui_assets.css' %}"/>
```

Comment:
We should also have a custom stylesheet we use, and put classes on relevant elements to control their styles. Using the style attribute causes a whole lot of redundancy and maintenance issues down the road since you have to make a change in many places rather than updating the styles for a particular selector.

Reply:
I'll make an issue for this

Reply:
Nevermind I see #36 does this